src/components/Post/Post.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User/User';
+import { Comments } from '../Comments/Comments';
+
+import './Post.css';
+
+export const Post = (props) => {
+  const { text,
+    title,
+    author,
+    emailAuthor,
+    address,
+    idx,
+    commentsList } = props;
+
+  return (
+    <div className="card text-white bg-primary">
+      <User author={author} email={emailAuthor} address={address} />
+      <div className="card-body">
+        <h4 className="card-title">{`#${idx} ${title}`}</h4>
+        <p className="card-text">{text}</p>
+      </div>
+      <h4 className="card-title">Comments:</h4>
+      <ul className="list-group">
+        {commentsList.map(({ name, id, postId, body, email }) => (
+          <Comments
+            key={`${name}-${id}-${postId}`}
+            className="list-group-item d-flex
+                  justify-content-between
```

Comment:
if you place one element on the new line, you have to do this with all your elements including the first one